docs/_documentations/troubleshooting.md

```diff
@@ -941,12 +941,16 @@ Check the Codewind output stream first when troubleshooting because it is partic
 
 If you report an issue, you will be asked to upload your logs.
 
-1. In VS Code, open **Help** > **Toggle Developer Tools**.
-2. Go to the **Console** tab.
-3. Enter *codewind.log* in the **Filter** box:
-<br>![Log location](images/cdt-vsc/logs-location.png)
+1. In VS Code, open the **Output** view.
+2. Select the Codewind output stream.
+3. The first line of the Codewind output stream is the location of the Codewind extension logs.
```

Comment:
Because the sentence, "The first line of the Codewind output...", isn't an action to be performed by the user, I suggest moving the sentence to step 2 instead:
"2. Select the Codewind output stream. The first line of the Codewind output stream is the location of the Codewind extension logs."